ui/dashboards/src/components/Variables/VariableList.tsx

```diff
@@ -45,14 +45,15 @@ export function TemplateVariableList() {
                 </TableRow>
               </TableHead>
               <TableBody>
-                {variableDefinitions.map((v) => (
-                  <TableRow key={v.name} sx={{ '&:last-child td, &:last-child th': { border: 0 } }}>
-                    <TableCell component="th" scope="row" sx={{ fontWeight: 'bold' }}>
-                      {v.name}
-                    </TableCell>
-                    <TableCell align="right">{v.kind}</TableCell>
-                  </TableRow>
-                ))}
+                {Array.isArray(variableDefinitions) &&
```

Comment:
U think it's worth leaving a comment to remove these checks after we've migrated dashboards? Will we migrate dashboards??

Trying to think of having less noise in the code, but also the comment itself could become noise